### PR TITLE
34 release notes added

### PR DESCRIPTION
**Pull request by CaroMac**

### CODE REVIEW

File: src/markdown-pages/highlights.md [hunk 1]
```diff
@@ -17,7 +17,20 @@ We have the following available Slack channels:
 
 Access the Galasa source code in [GitHub](https://github.com/galasa-dev) and open issues in the [project management repository](https://github.com/galasa-dev/projectmanagement).
 
-## 0.33.0 - Release Highlights
+
+## 0.34.0 - Release Highlights
+
+- Retrieve a list of all active personal access tokens in the Ecosystem by using the `galasactl auth tokens get` command.Use the information to revoke or delete an access token.
```
Comment (techcobweb) [inline src/markdown-pages/highlights.md:23]: Space needed between 'command' and 'Use' ?

File: src/markdown-pages/highlights.md [hunk 2]
```diff
@@ -17,7 +17,20 @@ We have the following available Slack channels:
...
+
+- The default etcd size in the Helm chart is increased from 1GB to 30GB and automatic history compaction is now turned on by default to minimise the risk of etcd running out of space.
```
Comment (techcobweb) [inline src/markdown-pages/highlights.md:25]: The default size of the PVC storage allocated by the Helm chart is increased...

**Review by techcobweb — APPROVED** [review]
Approved barring a few comments which need resolving. :-)

File: src/markdown-pages/highlights.md (hunk 2 above)
Comment (CaroMac) [reply src/markdown-pages/highlights.md]: have reworded

**Review by KirbyKatcher — APPROVED** [review]

**Pull request merged**